.travis.yml

```diff
@@ -34,6 +36,11 @@ jobs:
     - name: Unit testing
       stage: unit-test
       script: make unit-test
+    - name: Minikube test on amd64
+      stage: minikube-e2e-test
+      os: linux
+      arch: amd64
+      script: travis_wait 45 make setup minikube-test-e2e || travis_terminate 1
```

Comment:
We'll need to rerun the minikube tests during the rebuilds of releases (just like test-e2e). But since we are not rebuilding at the moment and also will be switching to OnePipeline, that can be added another time.